plugins/modules/identity/keycloak/keycloak_client.py

```diff
@@ -740,21 +752,15 @@ def main():
 
     module = AnsibleModule(argument_spec=argument_spec,
                            supports_check_mode=True,
-                           required_one_of=([['client_id', 'id']]))
+                           required_one_of=([['id', 'name'],
```

Comment:
That was slightly too much copy'n'paste ;-)
```suggestion
                           required_one_of=([['client_id', 'id'],
```